daily_log/tests/test_app.py

```diff
@@ -0,0 +1,6 @@
+# testing pytest in virtual env.
+# command used in terminal "pytest -m pytest"
+
+def test_test():
```

Comment:
I would hope this would of been an initial commit and you don't mean it's ready in which case: the workflow should be that you do a commit but for the merge request you edit the title with WIP: Unit testing using pytest so that I know that this is still work in progress. However since you moved the task in the Review column perhaps you consider this to be it.
Either way please do further investigation on how to test that the flask server is up and running and make a test that for the index where you can test that the page returns a status code of 200.